index.html

```diff
@@ -263,7 +292,7 @@ <h3 id="form-headline">
         <li class="footer-social-media-button"><a href="https://twitter.com/mperezbazan"><img src="./images/Twitter-dark.svg" alt="Twitter icon"/></a></li>
         <li class="footer-social-media-button"><a href="https://medium.com/@martin.perezbazan"><img src="./images/Medium-dark.svg" alt="Medium icon"/></a></li>
       </ul>
-      <hr class="footer-bottom-line"/>
```

Comment:
- [ ] Kindly make sure all the bottom social media icons are reading perfectly as right now some of them are broken to make the project more professional
![Screenshot from 2022-09-29 18-37-56](https://user-images.githubusercontent.com/83042742/193089640-882a1e6c-2d10-4048-aa56-c547617620fe.png)